### PR TITLE
Issue hint when failing to switch back to branch

### PR DESCRIPTION
**Pull request by Bert-R**
Switching back to the branch fails with the below error on a shallow checkout:

```
2023-04-18 21:34:51 [ERROR]   Failed to switch back to branch!
2023-04-18 21:34:51 [FATAL]   [fatal: reference is not a tree: 3afa21b5f9bef8e81396e3572a598d089e710b96]
```

With this change, it will look as follows:

```
2023-04-18 21:34:51 [ERROR]   Failed to switch back to branch!
2023-04-18 21:34:51 [INFO]   Check that you have the full git history, the checkout is not shallow, etc
2023-04-18 21:34:51 [INFO]  See https://github.com/github/super-linter#example-connecting-github-action-workflow
2023-04-18 21:34:51 [FATAL]   [fatal: reference is not a tree: 3afa21b5f9bef8e81396e3572a598d089e710b96]
```

Fixes #

## Proposed Changes

1. ...
2. ...
3. ...

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by Bert-R** [comment]
This is still actual

File: lib/functions/buildFileList.sh [hunk 1]
```diff
@@ -7,6 +7,13 @@
 ################################################################################
 ########################## FUNCTION CALLS BELOW ################################
 ################################################################################
+################################################################################
```
Comment (Hanse00) [inline lib/functions/buildFileList.sh:10]:
```suggestion

################################################################################
```

Comment (Hanse00) [reply lib/functions/buildFileList.sh:10]: I'd suggest you drop a newline in here to separate the above comment from the new one you're inserting.

**Review by Hanse00 — DISMISSED** [review]

File: lib/functions/buildFileList.sh [hunk 2]
```diff
@@ -7,6 +7,14 @@
 ################################################################################
 ########################## FUNCTION CALLS BELOW ################################
 ################################################################################
+
+################################################################################
+#### Function BuildFileList ####################################################
+function IssueHintForFullGitHistory() {
+    info "Check that you have the full git history, the checkout is not shallow, etc"
+    info "See https://github.com/github/super-linter#example-connecting-github-action-workflow"
```
Comment (Hanse00) [inline lib/functions/buildFileList.sh:15]:
```suggestion
  info "Check that you have the full git history, the checkout is not shallow, etc"
  info "See https://github.com/github/super-linter#example-connecting-github-action-workflow"
```

We use two whitespaces elsewhere for indentation.

**Review by Hanse00 — APPROVED** [review]

**Pull request merged**